### PR TITLE
Project completed

### PR DESCRIPTION
**Pull request by bouaik**
### Test javascript code with jest

### CODE REVIEW

File: src/analyse.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const analyse = (arr) => {
+  const analysed = {
+    average: average(arr),
+    min: min(arr),
+    max: max(arr),
+    length: arr.length,
+  };
+  return analysed;
+};
```
Comment (suretrust) [inline src/analyse.js]: - [x] **`Optional`** You could move `average`, `min`, and `max` into the `analyse` function. It's always better to use pure functions where possible.

```suggestion
const analyse = (arr) => {
  const average = (arr) => arr.reduce((acc, num) => acc + num, 0) / arr.length;
  const min = (arr) => Math.min(...arr);
  const max = (arr) => Math.max(...arr);
 
  const analysed = {
    average: average(arr),
    min: min(arr),
    max: max(arr),
    length: arr.length,
  };
  return analysed;
};
```

File: test/capitalized.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import capitalized from '../src/capitalized';
+
+test('it should return the String with first character capitalized', () => {
+  expect(capitalized('hello')).toBe('Hello');
```
Comment (suretrust) [inline test/capitalized.test.js:4]: - [x] **`Optional`** Let's add a test for an empty string. When writing tests, it's always good to think about all edge cases.

File: test/ceaser_cipher.test.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import cipher from '../src/ceaser_cipher';
+
+it('it should return a ciphered word', () => {
+  expect(cipher('abc', 1)).toBe('bcd');
```
Comment (suretrust) [inline test/ceaser_cipher.test.js:4]: - [x] Your test should have a case that wraps z to a. That way, you won't be missing out on anything.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @bouaik, 👋 

### Great job thus far! 🙌 Please see my comments. 👇 

- [x] I tagged some as `optional`, it would be nice to implement them as they make your codebase follow best practices, but they are not a requirement.

**KIndly submit another CRR once these issues have been solved.** _Happy coding!_ 🎉

**Review by shloch — APPROVED** [review]
Corrections made and project still working as well as tests

Keep up with the good work

HAPPY CODING
